internal/generator/generator.go

```diff
@@ -234,3 +241,21 @@ func adaptSelector(selector *kustomize.Selector) (output *kustypes.Selector) {
 	}
 	return
 }
+
+func isLocalRelativePath(path string) bool {
+	// From: https://github.com/kubernetes-sigs/kustomize/blob/84bd402cc0662c5df3f109c4f80c22611243c5f9/api/internal/git/repospec.go#L231-L239
+	// with "file://" removed/
+	for _, p := range []string{
+		// Order matters here.
+		"git::", "gh:", "ssh://", "https://", "http://",
+		"git@", "github.com:", "github.com/"} {
+		if len(p) < len(path) && strings.ToLower(path[:len(p)]) == p {
+			return false
+		}
+	}
+
+	if filepath.IsAbs(path) || filepath.IsAbs(strings.TrimPrefix(strings.ToLower(path), "file://")) {
```

Comment:
I think this is unneeded after looking at bit more at [the code](https://github.com/kubernetes-sigs/kustomize/blob/84bd402cc0662c5df3f109c4f80c22611243c5f9/api/internal/git/repospec.go#L130).
```suggestion
	if filepath.IsAbs(path) {
```

Reply:
I went full circle on this one, the current implementation is fine, as `filepath.IsAbs` is only checking the string, it isn't checking on on the fs.